### ISSUE TITLE
Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds

### ISSUE THREAD

**Issue by msftedad** [ng-bootstrap/ng-bootstrap#4566]
### Bug description:
Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds

Repro steps:
****
1. Launch [Angular powered Bootstrap (ng-bootstrap.github.io)](https://ng-bootstrap.github.io/#/components/datepicker/examples)
2. Tab to Date picker
3. Tab to Example and press enter.
4. Run AI4W and check for the results.

**Actual Results:**
Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds.

**Expected Result:**
Color contrast of the text with respect to background ratio should be greater than or equal to 4.5:1

### Link to minimally-working StackBlitz that reproduces the issue:
[Angular powered Bootstrap (ng-bootstrap.github.io)](https://ng-bootstrap.github.io/#/components/datepicker/examples)

### Versions of Angular, ng-bootstrap and Bootstrap:

ng-bootstrap: 15.1.1

Attachment:

**![Screenshot (126)](https://github.com/ng-bootstrap/ng-bootstrap/assets/93735775/410cf473-3549-4ebc-9f1c-fb6c762084ce)**

**Comment by msftedad** [comment]
Hi team, Any update on this issue?
